demo/kserve/deploy-remove.md

```diff
@@ -0,0 +1,145 @@
+# Deploy LLM model with Caikit+TGIS Serving runtime
+In this example we will be deploying a [flan-t5-small](https://huggingface.co/google/flan-t5-small) model with Caikit+TGIS Serving runtime. The model has already been containerized into S3 minio bucket.
+Before proceding, please ensure you have [installed](/docs/README.md) all pre-requisites, required operators and components.
+If you would like to use short scrips instead of following step-by-step manual, please see [here](#script-based-deployment-and-removal).
+
+Before proceeding, please make sure you are in the `/caikit-tgis-serving/demo/kserve/` directory.
+
+## Deploy Minio containing flan-t5-small model
+
+If you have your model in another S3-like object storage (e.g., AWS S3), you can skip this step.
+We will create a new project/namespace for minio and deploy it together with service account and data connection (secret with generated access key ID and secret access key). 
+~~~
+ACCESS_KEY_ID=THEACCESSKEY
```

Comment:
As this is just an example so we don't actually need to build ACCESS_KEY randomly. Any hardcoded key could work here as `ACCESS_KEY_ID=admin` and `SECRET_ACCESS_KEY=password`. With hardcoded value we don't need to do string manipulation and script to deploy minio would be such simpler.